presto-hive/src/main/java/com/facebook/presto/hive/HiveClientConfig.java

```diff
@@ -154,7 +154,7 @@
     private boolean isTemporaryStagingDirectoryEnabled = true;
     private String temporaryStagingDirectoryPath = "/tmp/presto-${USER}";
 
-    private boolean preloadSplitsForGroupedExecution;
+    private boolean useRewindableSplitSource;
```

Comment:
It would be easier to read if we had two commits, one that reverts `preloadSplitsForGroupedExecution`, second that introduces `useRewindableSplitSource`

Reply:
Will do. The side effect is that the test for preload splits would have to be removed and added back.

Reply:
Yeah, i also realized that. I don't think it's a problem. But if you don't like it - i don't feel particularly strong.